Adyen/UI/Form/Items/Error/FormErrorItemStyle.swift

```diff
@@ -0,0 +1,32 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import UIKit
+
+/// Contains the styling customization options for an error item in a form.
+public struct FormErrorItemStyle: ViewStyle {
+
+    /// The message style.
+    public var message = TextStyle(font: .preferredFont(forTextStyle: .body),
+                                   color: UIColor.Adyen.componentLabel,
+                                   textAlignment: .natural)
+
+    /// The corners style of the text item.
+    public var cornerRounding: CornerRounding = .fixed(6)
+
+    /// :nodoc:
+    public var backgroundColor = UIColor.Adyen.color(hex: 0xD10244).withAlphaComponent(0.1)
```

Comment:
maybe have this as a constant somewhere?

Reply:
I want merchants to be able to instantiate the style, then change the background color like this:

```
var style = FormErrorItemStyle()
style.backgroundColor = // ...
```

if I make it a constant, then I have to add it to the initializer, and the initializer will be bloated with many arguments.

Reply:
i meant the color, sorry :)

Reply:
👍 done